packages/terra-menu/src/Menu.jsx

```diff
@@ -53,12 +53,18 @@ const propTypes = {
    * Otherwise, the menu will display without an arrow and right aligned.
    */
   isArrowDisplayed: PropTypes.bool,
+  /**
+   * Header Title for menu.
+   * Header Title will only be visible if the menu contains at least one sub-menu.
+   */
+  headerTitle: PropTypes.string,
```

Comment:
Does every submenu display the same title? Should each submenu be able to supply a different title or is a single shared titled the requested behavior?

Reply:
The new `headerTitle` is specific to main-menu(first-tier). `sub-menus` header presentation title remains the same as it was, i,e same as its drilled in parents menu-item text.

Here is a minor change to the prop description 925d3978aa5c7b49ac4a66c4fd06ce07761016b9